### PR TITLE
Update launcher.ts to handle defaultViewport

### PR DESCRIPTION
**Pull request by mighty98**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/devtools/src/launcher.ts [hunk 1]
```diff
@@ -80,17 +80,17 @@ async function launchChrome (capabilities: ExtendedCapabilities) {
...
-        ],
         ...(headless ? [
             '--headless',
             '--no-sandbox'
         ] : []),
         ...chromeOptionsArgs
     ]
 
+    if(devtoolsOptions.defaultViewport !== null) {
+        chromeFlags.push(`--window-position=${DEFAULT_X_POSITION},${DEFAULT_Y_POSITION}`, `--window-size=${deviceMetrics?.width || DEFAULT_WIDTH},${deviceMetrics?.height || DEFAULT_HEIGHT}`)
```
Comment (christian-bromann) [inline packages/devtools/src/launcher.ts:91]:
```suggestion
        chromeFlags.push(
            `--window-position=${DEFAULT_X_POSITION},${DEFAULT_Y_POSITION}`,
            `--window-size=${deviceMetrics?.width || DEFAULT_WIDTH},${deviceMetrics?.height || DEFAULT_HEIGHT}`
        )
```

**Review by christian-bromann — COMMENTED** [review]
Do you think we can add a unit tests for this? Ideally somewhere in `packages/devtools/tests/launcher.test.ts`

**Review by christian-bromann — APPROVED** [review]
Awesome, LGTM 👍

**Comment by christian-bromann** [comment]
You can update the snapshots by calling `npx vitest -u`

**Comment by mighty98** [comment]
@christian-bromann I see test failures
image
I see the issues are caused due to the position of these variables.
I can change my code to put these in the same position as before. Should I?

**Comment by mighty98** [comment]
@christian-bromann Not sure if the failed tests are due to my changes.. I could not find any relations

**Comment by christian-bromann** [comment]
The build is currently broken which needs to get fixed. I will go ahead and merge this one. Thanks for all your work!

**Comment by mighty98** [comment]
Thank you @christian-bromann

**Pull request merged**